src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/get_accounts.py

```diff
@@ -0,0 +1,139 @@
+import json
+import logging
+import os
+import boto3
+from paginator import paginator
+from partition import get_partition
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+MANAGEMENT_ACCOUNT_ID = os.environ["MANAGEMENT_ACCOUNT_ID"]
+TARGET_OUS = os.environ.get("TARGET_OUS")
+REGION_DEFAULT = os.environ["AWS_REGION"]
+PARTITION = get_partition(REGION_DEFAULT)
+sts = boto3.client('sts')
+
+
+def main():
+    accounts = get_accounts()
+    with open('accounts.json', 'w') as outfile:
+        json.dump(accounts, outfile)
+
+    if TARGET_OUS:
+        accounts_from_ous = get_accounts_from_ous()
+        with open('accounts_from_ous.json', 'w') as outfile:
+            json.dump(accounts_from_ous, outfile)
+
+
+def list_organizational_units_for_parent(parent_ou):
+    organizations = get_boto3_client('organizations', f'arn:{PARTITION}:sts::{MANAGEMENT_ACCOUNT_ID}:role/OrganizationAccountAccessRole-readonly', 'getOrganizationUnits')
```

Comment:
Can you make the `OrganizationAccountAccessRole` use the role name defined in adfconfig.yml instead.
You can retrieve that using the SSM Parameter Store `cross_account_access_role` key at the moment.

Reply:
Read parameter from SSM Parameter Store